src/Microsoft.Framework.BufferEntryCollection.Internal/BufferEntryCollection.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Framework.Internal
+{
+    /// <summary>
+    /// An ordered collection of strings with an enumerator that iterates over it.
+    /// </summary>
+    internal class BufferEntryCollection : IEnumerable<string>
```

Comment:
Make sense. Thanks for the clarification. I do think the namespace can be more specific though.